srcpkgs/lzfse/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'lzfse'
+pkgname=lzfse
+version=1.0
+revision=1
+build_style=cmake
+short_desc="LZFSE compression library and command line tool"
+maintainer="dkwo <npiazza@disroot.org>"
+license="BSD-3-Clause"
+homepage="https://github.com/lzfse/lzfse"
+distfiles="https://github.com/lzfse/lzfse/archive/refs/tags/lzfse-${version}.tar.gz"
+checksum=cf85f373f09e9177c0b21dbfbb427efaedc02d035d2aade65eb58a3cbf9ad267
+
+lzfse-devel_package() {
+	short_desc+=" - development files"
+	depends="${sourcepkg}>=${version}_{revision}"
+	pkg_install() {
+		vmove usr/include/lzfse.h
+		vmove usr/lib/liblzfse.so
+	}
+}
+
+post_install() {
```

Comment:
post_install should go before subpackages

Reply:
done.